### PR TITLE
Reside 357: Add anmol to Blog 

### PR DESCRIPTION
**Pull request by absternator**
- Add anmol to blog 
-  move lekan to alumni

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -44,3 +44,4 @@ Mantra studied Mathematics at university and realised most of it won't be used a
 * [Marie Gronemeyer](https://github.com/muppi1993)
 * [Nicolas Dolan](https://github.com/Nicolas-Dolan)
 * [Alex Hill](https://github.com/hillalex)
+* [Lekan Anifowoshe](https://github.com/carbowizzy)
```
Comment (EmmaLRussell) [inline content/about.md]: Lekan actually updated his github id but didn't change it here !

```suggestion
* [Lekan Anifowoshe](https://github.com/LekanAnni)
```

**Review by EmmaLRussell — COMMENTED** [review]
Looks great, just a small suggestion to update Lekan's link. 

For future reference, we'd usually just name the branch after the ticket number for brevity, e.g. `reside-357`

File: content/about.md (hunk 1 above)
Comment (absternator) [reply content/about.md]: updated

**Review by EmmaLRussell — APPROVED** [review]
Thanks!

**Pull request merged**